### PR TITLE
Remove the Validate always section in validator docs

### PR DESCRIPTION
**Pull request by adriangb**
Selected Reviewer: @lig

### CODE REVIEW

**Comment by adriangb** [comment]
please review

File: docs/usage/validators.md [hunk 1]
```diff
@@ -321,37 +321,6 @@ print(Model(a='invalid').a)
 #> 2000-01-01 00:00:00
 ```
 
-## Validate always
-
-**TODO this content is wrong!**
-
-For performance reasons, by default validators are not called for fields when a value is not supplied.
-However there are situations where it may be useful or required to always call the validator, e.g.
-to set a dynamic default value.
```
Comment (ybressler) [inline docs/usage/validators.md:330]: I would argue it is important to keep the section, but describe behavior instead. You can use `validate_defaults` to perform validation always...

Comment (adriangb) [reply docs/usage/validators.md:330]: The reality of the matter is that (1) these docs as they currently are do not help and (2) many of the requests we get for this feature either are better solved in another manner or have big drawbacks. We can re-add docs when we find a simple example that this is the right fit for, once we get actual users requesting this that can only be solved by this option, etc.

Comment (ybressler) [reply docs/usage/validators.md:330]: Got it. Valid.

**Review by lig — APPROVED** [review]
👌🏻 🤷🏻‍♂️

**Pull request merged**